mitodl/ocw-studio
# Merge add and edit content forms

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #191 
Fixes #154 

#### What's this PR do?
The forms and pages for editing and adding content are merged. Both will use a drawer according to the mockup in #191

#### How should this be manually tested?
 - Editing content should work exactly like before.
 - When you click the "Add Resource" or "Add Page" button, you should see a drawer pop out instead of being directed to a new page like before.
   - If you leave the title blank it should fail validation.
   - When you click 'Save' you should see the new item show up on the listing.

#### Screenshots
![Screenshot from 2021-04-09 09-12-28](https://user-images.githubusercontent.com/863262/114185598-e7f5b500-9913-11eb-859e-da731cbfc1e6.png)
![Screenshot from 2021-04-09 09-12-48](https://user-images.githubusercontent.com/863262/114185600-e88e4b80-9913-11eb-90b2-d0934518624a.png)


![Screenshot from 2021-04-09 09-14-16](https://user-images.githubusercontent.com/863262/114185717-078cdd80-9914-11eb-8a7e-1d06674cc010.png)
![Screenshot from 2021-04-09 09-14-30](https://user-images.githubusercontent.com/863262/114185718-078cdd80-9914-11eb-9820-b1ad724b8c4f.png)
